docs/track.rst

```diff
@@ -3056,6 +3056,39 @@ The following meta data is always returned:
 * ``weight``: The number of fetched pages. Should always equal to the ``pages`` parameter.
 * ``unit``: The unit in which to interpret ``weight``. Always "ops".
 
+downsampling
+~~~~~~~~~~~~~~~~~~~
+
+Executes a downsampling operation on an index producing a new index whose data is aggregated on the @timestamp field.
+
+
+Properties
+""""""""""
+
+* ``body`` (mandatory): The ``body`` property must contain the ``fixed_interval`` key defined as in `https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-datehistogram-aggregation.html#fixed_intervals` the ``source-index`` and the ``target-index``. The source index is an existing index which includes a ``@timestamp`` field. Write operations to the source index must be disabled. Tne target index is a new index that is created by the downsamplig operation and including aggregated data.
```

Comment:
Most of the time we abstract common or critical `body` elements for the user to explicit parameters. If the `body` has required keys, it's likely better to just parameterize the operation instead of providing the body verbatim. This also gives the operation advantage over just using a `raw-request`.

Reply:
I was thinking about that and at the end I chose to have the body because I had the feeling that having the body was "more consistent" with other operations. But I see your point here and I will get rid of it.